packages/webpack/tsconfig.json

```diff
@@ -6,5 +6,9 @@
     "checkJs": true,
     "rootDir": "src",
     "outDir": "types",
+    "paths": {
+      "lavamoat-core": ["../core"]
+    }
   },
+  "references": [{ "path": "../core/tsconfig.json" }]
```

Comment:
Error cause seems to require ES2022 and we're targeting earlier. 

I think since we're dropping node 16 we can update https://github.com/LavaMoat/LavaMoat/blob/25405141525e75c214f553e4c082c6b2ed79cfdd/.config/tsconfig.workspace.json to extend node18 not node16 by default and narrow down in packages we want to keep node16 support.

I'll override in webpack for this branch and see if it helps.

Reply:
Seems to work. 

Now we know dropping node16 is not done until we depend on @tsconfig/node18

And the way our tsconfig is set up doesn't easily allow dropping support selectively.